proposals/instructor-training-selection.md

```diff
@@ -0,0 +1,144 @@
+# Instructor Training Selection
+
+**Warning: This is only a draft.**
+
+**Warning: This only applies for the two-day training course in December 2015.**
+
+## Terminology
+
+-   An **instructor** is someone who teaches a workshop. Instructor training (IT)
+    prepares them to do this.
+
+## Why Select Attends for Instructor Training
+
+SCF/DC wants to train as more instructors as possible
+but demand for instructor training is higher than what SCF could offer
+at the moment.
+
+## Eligibility
+
+### Groups
+
+-   Be able to Co-locate in one room for the two days
+-   Commit to jointly running one workshop no later than April 1st, 2016.
```

Comment:
I suggest to change this to:

We expect all students to gather in groups that are able to co-locate in one room for the duration of the Instructor Training. Each group should commit to jointly run one workshop no later than April 1st, 2016. Please note: how to run a workshop is covered in the curriculum.